notes/health-care-evaluation.vbs

```diff
@@ -3997,360 +3948,664 @@ If HC_form_name = "No Form - Ex Parte Determination" Then
-
-					'Validation to ensure that ExParte field updated to N
-					If check_HC_ex_parte_determination <> "N" THEN err_msg = err_msg & vbCr & "* You must enter 'N' for ExParte."
-
-					'Validate that ExParte Renewal Month is correct
-					'TO DO - add validation to ensure that date updated in HC renewal screen is the same as date provided in SQL table
-					If check_ex_parte_renewal_month = "__ ____" THEN err_msg = err_msg & vbCr & "* You must enter the month and year for the Ex Parte renewal month."
-
-					'Error message handling
-					IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
-				Loop until err_msg = ""
-					'Add to all dialogs where you need to work within BLUEZONE
-					CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
-			LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
-		End If
-	End If
-	' MsgBox "STOP HERE FOR NOW!!!"
+				'Add validation to ensure that ex parte denial explanation is not greater than 255 characters
+				If len(ex_parte_denial_explanation) > 255 Then err_msg = err_msg & vbCr & "* The explanation for the Ex Parte denial is too long and should be shortened. The length of the information cannot be more than 255 characters."
 
-	If ex_parte_determination = "Appears Ex Parte" Then appears_ex_parte = True
-	If ex_parte_determination = "Cannot be Processed as Ex Parte" Then appears_ex_parte = False
-	If ex_parte_determination = "Health Care has been Closed" Then appears_ex_parte = False
-	If ex_parte_determination = "Case Transfered Out of County" Then appears_ex_parte = False
-	' MsgBox "appears_ex_parte - " & appears_ex_parte
+				'Add validation to ensure phase 1 changes summary field is not blank
+				If trim(phase_1_changes_summary) = "" Then err_msg = err_msg & vbCr & "* You must provide a summary of any changes since the Phase 1 determination for this case."
 
-	If user_ID_for_validation <> "CALO001" AND user_ID_for_validation <> "MARI001" Then
-		' MsgBox "STOP - YOU ARE GOING TO UPDATE"
-		sql_format_ex_parte_denial_explanation = replace(ex_parte_denial_explanation, "'", "")
-		objUpdateSQL = "UPDATE ES.ES_ExParte_CaseList SET SelectExParte = '" & appears_ex_parte & "', Phase1HSR = '" & user_ID_for_validation & "', ExParteAfterPhase1 = '" & ex_parte_determination & "', Phase1ExParteCancelReason = '" & sql_format_ex_parte_denial_explanation & "' WHERE CaseNumber = '" & SQL_Case_Number & "'"
+				'Add validation to ensure that phase 1 changes summary field is not greater than 255 characters
+				If len(phase_1_changes_summary) > 255 Then err_msg = err_msg & vbCr & "* The summary of changes since the Phase 1 determination is too long and should be shortened. The length of the information cannot be more than 255 characters."
```

Comment:
The length limit is for adding this information to the SQL table. We only have one field for these changes, so I think we do not need to test the length of the phase_1_changes_summary or phase_2_notes. 

I think we can use just the information in the denial_explanation field for adding detail to the SQL able. 

Or do you think we should do a combination of details? In that case we need to create a new variable and combine the correct notes and ensure that entire thing is under 255.

I think any argument could be made for any of these options BUT we can only output one variable to the notes and that has a character limit of 255. 

Also, these variables should all be trimmed first, since we need a trimmed variable to see if it is blank AND a trimmed variable to test the length AND the variable should be trimmed before entry into CASE/NOTE. 

(Also I should stop writing sentences with the same capitalization syntax as my code.)

Reply:
Updated - I trimmed each of the fields within the start of the dialog. The denial explanation and phase 1 changes summary fields cannot be left blank.